lib/block_elements.ex

```diff
@@ -123,11 +123,17 @@ defmodule BlockBox.BlockElements do
 
   ## Options
   Options are not included by default.
-  * `:initial_option` - `t:BlockBox.CompositionObjects.option_object/0`
+  * `:initial_option` - `t:BlockBox.CompositionObjects.option_object/0` | `t:integer/0` can provide a option object or an index
```

Comment:
I made the decision to not include links to primitive types because if we did that for all primitive types, the docs would quickly become ugly and harder to read. Also, since this is in the documentation and not in a spec, just use the word "or".

Reply:
I don't think it will make the docs ugly, it will just help point users to the appropriate type. It would be ugly if it added a bunch of additional text or something. Regardless, I understand that the meaning of primitive types should be implied, if you're using elixir you should have an understanding of what an integer is and that it is a primitive type.

Reply:
every time your mouse hovers over a type link like these, hex docs will open a pop up that shows you the type definition and obscures whatever is behind it on the page. Given most of our options accept primitive types, reading the docs would become a landmine of popups that show useless information.